connect-widget/app/src/pages/connect/index.tsx

```diff
@@ -34,7 +35,9 @@ const ConnectPage = () => {
         setAgreementState(setAgreement);
         //eslint-disable-next-line @typescript-eslint/no-explicit-any
       } catch (err: any) {
-        setError(err);
+        if (err.message !== DemoTokenError.DEFAULT_MSG) {
+          setError(err);
+        }
         capture.error(err, { extra: { context: `connect.setup` } });
```

Comment:
I know I said demo token _could_ be reported to Sentry, but I'm thinking we _shouldn't_:
- its just for our usage;
- if someone figures it out and uses it as well the worse I can see happening is they reaching out for support because they don't notice/forget the widget is running in demo mode.

Thoughts, @Goncharo ?

Reply:
Perhaps, it's better to show a message on screen without a pop up to indicate demo mode? Kinda like what we have for sandbox on the dash?

Reply:
Agreed - there's no point I can see of reporting Demo events to Sentry

Reply:
Fixed!